File: CafeEase/CafeEase.Model/Payment.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CafeEase.Model
{
    public class Payment
    {
        public int Id { get; set; }
        public string? Method { get; set; }
        public string? Status { get; set; }
        public int OrderId { get; set; }
    }
}

File: CafeEase/UI/cafeease_mobile/lib/models/payment.dart
import 'package:json_annotation/json_annotation.dart';

part 'payment.g.dart';

@JsonSerializable()
class Payment {
  int? id;
  String? method;
  String? status;
  int? orderId;

  Payment({this.id, this.method, this.status, this.orderId});

  factory Payment.fromJson(Map<String, dynamic> json) =>
      _$PaymentFromJson(json);

  Map<String, dynamic> toJson() => _$PaymentToJson(this);
}
